README.md

```diff
@@ -6,6 +6,12 @@ and update etcdclient to v3
 
 deprecates 17media/go-etcd
 
+note that we ignore errors that files are too large to push (limitation for server side: 1.5 MiB, client side: 4 MiB), because they are not etcd meant to maintain
```

Comment:
It seems the default value is 2MB for client-side.
```
If not specified, client-side send limit defaults to 2 MiB (1.5 MiB + gRPC overhead bytes) and receive limit to math.MaxInt32. Please see [clientv3 godoc](https://pkg.go.dev/github.com/etcd-io/etcd/clientv3#Config) for more detail.
```
https://github.com/etcd-io/etcd/blob/08407ff7600eb16c4445d5f21c4fafaf19412e24/client/v3/config.go#L46